src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -872,6 +905,11 @@ public static TimestampGranularity get(JdbcSourceConnectorConfig config) {
     }
   }
 
+  public enum TransactionIsolationMode {
+    DEFAULT, READ_UNCOMMITTED, READ_COMMITTED,
```

Comment:
Also, consider implementing this similar to how `numeric.mapping` is implemented so that you don't need to keep doing `TransactionIsolationMode.valueOf` to go from the specified value back to the enum.

Reply:
Pushing back on this because after the changes that were suggested was added, we only call `valueOf` once and the enum extension seems like overkill for it